SingularityUI/app/views/expandableTableSubview.coffee

```diff
@@ -51,6 +51,7 @@ class ExpandableTableSubview extends View
         @$el.html @template
             synced:  @collection.synced
             data:    _.pluck @collection.models, 'attributes'
+            config: config
```

Comment:
one thing that I probably didn't make clear enough (2nd to last bullet point in https://github.com/HubSpot/Singularity/pull/560#issuecomment-106590733) is we need to interpolate `TASK_ID` in the log path filename(s) to cover the situation where the Singularity executor is used. I'd suggest specifically using the string `$TASK_ID` for interpolation. This makes things a little hairier because we probably don't want to be doing this for every usage of `ExpandableTableSubview`, so i'd consider adding a `getRenderData()` method that subclasses can override with custom data